src/base/preferences.cpp

```diff
@@ -1477,6 +1477,21 @@ void Preferences::setUpdateCheckEnabled(const bool enabled)
 }
 #endif
 
+#ifdef Q_OS_MACOS
+bool Preferences::isSpeedInDockEnabled() const
+{
+    return value(u"Preferences/Speed/ShowSpeedInDock"_s, true);
```

Comment:
Why is "Preferences/Speed/"? Are there other speed-related settings that are supposed to be grouped together with this one?